presto-native-execution/scripts/setup-centos.sh

```diff
@@ -14,7 +14,7 @@
 set -e
 set -x
 
-export FB_OS_VERSION=v2022.11.14.00
+export FB_OS_VERSION=v2023.12.04.00
```

Comment:
I remember asking why we don't update these in other scripts, only in macos.

Reply:
I misunderstood your comment :). Apologies!

Reply:
Got it. Thank you for clarifying.